src/coffee/repeater-task-queue.coffee

```diff
@@ -0,0 +1,73 @@
+{ Repeater } = require 'sphere-node-utils'
+Promise = require 'bluebird'
+util = require 'util'
+TaskQueue = require './task-queue'
+
+# Response messages which will be handled by `RepeaterTaskQueue`
+retryKeywords = [
```

Comment:
@emmenko Some examples where status code is not defined:

```
{
    "error": "{ [HttpError: getaddrinfo EAI_AGAIN api-at-rackspace.sphere.io:443]\n  message: 'getaddrinfo EAI_AGAIN api-at-rackspace.sphere.io:443',\n  body: \n   { statusCode: undefined,\n     message: 'getaddrinfo EAI_AGAIN api-at-rackspace.sphere.io:443',\n     originalRequest: \n      { endpoint: '/channels?where=key%3D%22orderexport%22',\n        options: \n         { config: { project_key: 'xxx' },\n           host: 'api-at-rackspace.sphere.io',\n           protocol: 'https',\n           timeout: 360000,\n           rejectUnauthorized: true,\n           headers: \n            { 'User-Agent': 'xxx-cli-0.13.4',\n              Authorization: 'Bearer **********' },\n           uri: 'https://api-at-rackspace.sphere.io/xxx' } } },\n  name: 'HttpError',\n  code: undefined }",
    "hostname": "xxx",
    "level": 50,
    "msg": "'order-export' - Unexpected error after '0d 0h 0m 5.297s'",
    "name": "xxx-cli-0.13.4",
    "pid": 1,
    "time": "2016-09-13T18:25:09.745Z",
    "v": 0
}
```

```
{
    "error": "Error: read ECONNRESET\n    at exports._errnoException (util.js:907:11)\n    at TCP.onread (net.js:557:26)",
    "hostname": "xxx",
    "level": 50,
    "msg": "Error details: ",
    "name": "xxx-cli-0.12.9",
    "pid": 1,
    "time": "2016-09-06T00:46:01.203Z",
    "v": 0
}
```

We could make this list as part of constructor which might be overridden if defined in options?